DXRFixes/DeusEx/Classes/LaserTrigger.uc

```diff
@@ -23,7 +23,33 @@ function BeginAlarm()
             A.Trigger(Self, Pawn(emitter.HitActor));
 }
 
+function EndAlarm()
+{
+    super.EndAlarm();
+    LightType = LT_None;
+}
+
+function Tick(float deltaTime)
+{
+    Super.Tick(deltaTime);
+
+    if (AmbientSound != None){ //Alarm is happening
+        // flash the light and texture
+        if ((Level.TimeSeconds % 0.5) > 0.25)
+        {
+            LightType = LT_Steady;
```

Comment:
you can't just use `LT_Blink` with `LightPeriod=0.5` or something? 

I see this is how vanilla AlarmPanels do it lol